client/my-sites/plugins-wpcom/jetpack-plugins-panel.jsx

```diff
@@ -140,6 +155,8 @@ class JetpackPluginsPanel extends Component {
 			} ) )
 			.filter( group => group.plugins.length > 0 );
 
+		const browserUrl = '/plugins/browse' + ( this.props.siteSlug ? '/' + this.props.siteSlug : '' );
```

Comment:
could probably get by with a more declarative approach here…

```js
const browserUrl = [ '/plugins/browse', this.props.siteSlug ].join( '/' );
```

would want to confirm before changing